packages/vue-component-library/src/styles/ftva/_button-dropdown.scss

```diff
@@ -1,4 +1,6 @@
 .ftva.button-dropdown {
+    z-index: 10;
```

Comment:
@angelahuqing Same here. I think an index value of 1 should be okay. Don't recall why I initially set it so high to begin with! Test locally to be sure.